package.json

```diff
@@ -2,17 +2,21 @@
   "name": "helloreact",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:3000",
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.0.5",
+    "react-router-dom": "^6.10.0",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3001 react-scripts start",
```

Comment:
- I get an error when I try running your code. If you are trying to set the port for application, please do this:
`"start": "set PORT=3001 && react-scripts start",` Please fix this 🙏

Reply:
- [ ] Actually `"start": "set PORT=3001 && react-scripts start"` this set up is for the windows users I'm using mac OS so for me this works `"PORT=3001 react-scripts start",` if you are using a different machine then you'll have to setup for your machine manually